File: Java/IfElseLadder.java
//Java program to display Grade using if-else ladder

import java.util.Scanner;

class IfElseLadder
{
public static void main(String args[])
{
Scanner sc = new Scanner(System.in);
int marks;

System.out.println("Enter Your Marks:"); //Ask user to enter marks
marks = sc.nextInt();

//if-else ladder to calculate grade
if(marks >= 90){
System.out.println("Grade: A+");
}else if(marks >= 80){
System.out.println("Grade: A");
}else if(marks >= 70){
System.out.println("Grade: B");
}else if(marks >= 60){
System.out.println("Grade: C");
}else if(marks >= 50){
System.out.println("Grade: D");
}else if(marks >= 35){
System.out.println("PASS");
}else{
System.out.println("Better Luck Next Time");
}

}
}

File: Kotlin/IfElseLadder.kt
//Kotlin program to display Grade using if-else ladder

fun main()
{
println("Enter Your Marks:") //Ask user to enter marks
var marks = readLine()!!.toInt()

//if-else ladder to calculate grade
if(marks >= 90){
println("Grade: A+")
}else if(marks >= 80){
println("Grade: A")
}else if(marks >= 70){
println("Grade: B")
}else if(marks >= 60){
println("Grade: C")
}else if(marks >= 50){
println("Grade: d")
}else if(marks >= 35){
println("PASS")
}else{
println("Better Luck Next Time")
}

}